### PR TITLE
[code42-636] Improve documentation examples

### PR DESCRIPTION
**Pull request by unparalleled-js**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Improves some documentation 

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [x] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [n/a] Tests
- [x] Documentation 

## Demisto Partner?
- [x] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

File: Packs/Code42/Integrations/Code42/Code42.yml [hunk 1]
```diff
@@ -805,7 +805,7 @@ script:
     - contextPath: Code42.DepartingEmployee.DepartureDate
       description: The departure date for the Departing Employee.
       type: Unknown
-  dockerimage: demisto/py42:1.0.0.9806
+  dockerimage: demisto/py42:1.0.0.10082
```
Comment (unparalleled-js) [inline Packs/Code42/Integrations/Code42/Code42.yml:808]: I am not sure why some of these changes are duplicated

Comment (guykeller) [reply Packs/Code42/Integrations/Code42/Code42.yml:808]: What do you mean?

File: Packs/Code42/Integrations/Code42/Code42_test.py [hunk 2]
```diff
@@ -802,8 +802,8 @@
                 "filterClause": "AND",
                 "filters": [
                     {"operator": "IS_NOT", "term": "exposure", "value": "IsPublic"},
-                    {"operator": "IS_NOT", "term": "exposure", "value": "SharedViaLink"},
                     {"operator": "IS_NOT", "term": "exposure", "value": "OutsideTrustedDomains"},
+                    {"operator": "IS_NOT", "term": "exposure", "value": "SharedViaLink"},
```
Comment (guykeller) [inline Packs/Code42/Integrations/Code42/Code42_test.py:806]: Why was this change necessary?

File: Packs/Code42/Integrations/Code42/Code42.yml (hunk 1 above)
Comment (unparalleled-js) [reply Packs/Code42/Integrations/Code42/Code42.yml:808]: This change is already present in the base branch... https://github.com/demisto/content/pull/8282

File: Packs/Code42/Integrations/Code42/Code42_test.py (hunk 2 above)
Comment (unparalleled-js) [reply Packs/Code42/Integrations/Code42/Code42_test.py:806]: Look like auto-formatting ordering the dict values? I am not sure. This change is already in the base branch and I don't think I made it. I just want the readme changes in, and we already have a PR open, so that is why I targeted that branch

**Comment by reutshal** [comment]
@guykeller This branch has changed taken from https://github.com/demisto/content/pull/8282, which was already merged to the internal branch (still not in master).
Once it will be merged to master you will no longer see the changes

**Comment by guykeller** [comment]
Could you please merge from the base branch into the current branch? Some of the changes were already merged in a different PR.

**Comment by unparalleled-js** [comment]
> Could you please merge from the base branch into the current branch? Some of the changes were already merged in a different PR @unparalleled-js

Done!

**Comment by guykeller** [comment]
@unparalleled-js 
Since this was split into two PRs, we will need to treat this change as a new version.
Could you please use the SDK to bump the version & add a new release notes file?
